TxLib/Fluent/TxExtensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace Unclassified.TxLib.Fluent
+{
+    /// <summary>
+    ///     Provides fluent builder for Tx API
+    /// </summary>
+    public static class TxExtensions
+    {
+        #region Static members
+
+        /// <summary>
+        ///     Extension method for strings to create <see cref="TxFluentBuilder" />
+        /// </summary>
+        /// <param name="key">Text key to search.</param>
+        /// <returns>New TxFluentBuilder instance</returns>
+        public static TxFluentBuilder Tx(this string key)
+        {
+            return new TxFluentBuilder(key ?? string.Empty);
+        }
+
+        /// <summary>
+        ///     Extension method for strings to create <see cref="TxFluentBuilder" />. Ignores key case.
+        /// </summary>
+        /// <param name="key">Text key to search.</param>
+        /// <returns>New TxFluentBuilder instance</returns>
+        public static TxFluentBuilder Txi(this string key)
```

Comment:
Yep u are right this method is too specific. Soon I will start to improve your editor for my cases. So it will be a lot of pull requests I guess.